packages/router/src/lib/models.ts

```diff
@@ -4,6 +4,7 @@ import {
   CanActivateFn,
   CanDeactivateFn,
   CanMatchFn,
+  DeprecatedGuard,
```

Comment:
What version of Angular was this introduced?

Reply:
I think it was added since Angular 15.2

https://github.com/angular/angular/commit/926c35f4ac70f5e4d142e545d6d056dd67aac97b

Reply:
Ok cool. We support back to Angular v15 so just wanted to make sure